test/jsonapi/plugs/query_parser_test.exs

```diff
@@ -132,11 +132,11 @@ defmodule JSONAPI.QueryParserTest do
   test "parse_pagination/2 turns a fields map into a map of pagination values" do
     config = struct(Config, view: MyView)
     assert parse_pagination(config, config.page).page.__struct__ == JSONAPI.Page
-    assert parse_pagination(config, %{"limit" => 1}).page.limit == 1
-    assert parse_pagination(config, %{"offset" => 1}).page.offset == 1
-    assert parse_pagination(config, %{"page" => 1}).page.page == 1
-    assert parse_pagination(config, %{"size" => 1}).page.size == 1
-    assert parse_pagination(config, %{"cursor" => 1}).page.cursor == 1
+    assert parse_pagination(config, %{"limit" => "1"}).page.limit == 1
+    assert parse_pagination(config, %{"offset" => "1"}).page.offset == 1
+    assert parse_pagination(config, %{"page" => "1"}).page.page == 1
+    assert parse_pagination(config, %{"size" => "1"}).page.size == 1
+    assert parse_pagination(config, %{"cursor" => "1"}).page.cursor == 1
```

Comment:
Does it make sense to add a case like `parse_pagination(config, %{"page" => "boogers"})`? This case would expose `String.to_integer/1` throwing when the String doesn't represent an Integer.

Reply:
Yes, also, cursor may rightfully be a string, not an integer.